src/Pages/LoginPage/LoginPage.js

```diff
@@ -1,27 +1,12 @@
 import LoginForm from "Components/LoginForm/LoginForm";
-import { Navigate } from "react-router";
-import useAuthentication from "Services/firebase/useAuthentication";
 import useTitle from "Utils/useTitle";
 
 const LoginPage = function () {
   useTitle("Bragi 3000 - Login");
 
-  const { authReady, loggedIn } = useAuthentication();
-
-  if (!authReady)
-    return (
-      <span>Waiting for auth...</span>
-    );
-
-  if (loggedIn)
-    return (
-      <Navigate to="/app" />
-    );
-
   return (
     <div>
       <h1>Login</h1>
-
       <LoginForm />
```

Comment:
I was wondering about password resetting in case a user forgot his password. What happens in that case? As in `Components/SignUpForm` there is no way to handle it. Should we create an issue to handle this later as it's not high priority

Reply:
Yeah, feel free to create an issue. I know Firebase has some options for this and email verification at sign-up, so it should be quite easy to implement at some point.